setup/install_gatling/30_classification/50_classify_nodes_via_site_pp.rb

```diff
@@ -2,7 +2,7 @@
 skip_test 'Installing PE, not FOSS' unless ENV['BEAKER_INSTALL_TYPE'] == 'foss'
 
 def generate_sitepp()
-  "node /.*agent.*/ { include #{ENV['PUPPET_SCALE_CLASS']} }\nnode /#{agent.hostname}/ { include #{ENV['PUPPET_SCALE_CLASS']} }\nnode 'default' {}"
+  "node /.*agent.*/ { include #{ENV['PUPPET_SCALE_CLASS']} }\nnode 'default' {}"
```

Comment:
i realize this was already like this.  could this be put in a heredoc or something more readable which would reproduce the newlines more effectively?